libraries/AP_Scripting/AP_Scripting.h

```diff
@@ -47,6 +51,24 @@ class AP_Scripting
         bool session;
     } terminal;
 
+    struct mavlink_msg {
+        mavlink_message_t msg;
+        mavlink_channel_t chan;
+    };
+
+    struct mavlink_output {
+        char data[256]; // maximum payload size
+        int32_t msgid;
+        mavlink_channel_t chan;
+    };
+
+    struct mavlink {
+        ObjectBuffer<struct mavlink_msg> * input;
+        ObjectBuffer<struct mavlink_output> * output;
+        int32_t accept[16];
```

Comment:
This is surprisingly low.

Reply:
It starts to eat a lot of memory, and the linear search becomes hard to defend if increase this much. I'm inclined to leave this small until users start to complain. It also seems to be likely that if we increase this much we'd need to increase the buffer for handling the messages, which is a large memory increase.